src/domain/constants/prizeMap.js

```diff
@@ -0,0 +1,8 @@
+export const PRIZE_MAP = {
+  FIRST: 2000000000,
+  SECOND: 30000000,
+  THIRD: 1500000,
+  FOURTH: 50000,
+  FIFTH: 5000,
+  LOSS: 0,
+};
```

Comment:
```suggestion
export const PRIZE_MAP = {
  FIRST: 2_000_000_000,
  SECOND: 30_000_000,
  THIRD: 1_500_000,
  FOURTH: 50_000,
  FIFTH: 5_000,
  LOSS: 0,
};
```

이렇게 표현하면 숫자를 볼 때 더 편하지 않을까 싶어요!

Reply:
오 처음 알게된 문법이네요 감사합니다!